bigbone/src/main/kotlin/social/bigbone/MastodonClient.kt

```diff
@@ -284,9 +290,23 @@ private constructor(
             this.debug = true
         }
 
+        private fun getInstanceVersion(): String {
+            val httpClient = OkHttpClient.Builder().build()
+            val v2InstanceRequest = httpClient.newCall(Request.Builder().url("$baseUrlV2/instance").get().build())
```

Comment:
We could use `MastodonClient.fullUrl(instanceName, pathV1)` here and in the next line to make sure that we're building the URL like anywhere else.

Reply:
Fixed 🙂